src/components/scenes/SendConfirmationScene.js

```diff
@@ -87,6 +89,25 @@ export class SendConfirmation extends Component<Props, State> {
   constructor (props: Props) {
     super(props)
     slowlog(this, /.*/, global.slowlogOptions)
+
+    const { guiMakeSpendInfo } = props
+    // Flow Errors when checking the index - muted the flow error
+    // Test errors also on guiMakeSpendInfo
+    const spendTarget = guiMakeSpendInfo.spendTargets[0]
+    const nativeAmount = spendTarget.nativeAmount
+
+    // Checks for native amount
+    const checkNativeAmount = () => {
```

Comment:
Why make this a function. Just make it inline code and assign to `isFiatOnTop` and `isFocus`